xml/api/attrd_updater-2.18.rng

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't think we should have separate schemas for crm_attribute and attrd_updater. The APIs are meant to be logical and not tied one-to-one to tools. We can have a single attribute schema with all the possible variations from both commands (which hopefully will be as unified as possible).

Reply:
This was actually my first thought on this before I saw that attrd_updater can show multiple attributes at once while crm_attribute can only show one.  I've merged them back into the same schema.